src/components/Editor/index.js

```diff
@@ -745,8 +754,9 @@ Editor.propTypes = {
   toggleBreakpoint: PropTypes.func.isRequired,
   addOrToggleDisabledBreakpoint: PropTypes.func.isRequired,
   toggleDisabledBreakpoint: PropTypes.func.isRequired,
-  isEmptyLine: PropTypes.func,
-  continueToHere: PropTypes.func
```

Comment:
@hughugsy Any reason for removing this line?

Reply:
Sorry that's mistake:) I'll fix that